package.json

```diff
@@ -58,5 +58,9 @@
         "extends": [
             "@commitlint/config-conventional"
         ]
+    },
+    "dependencies": {
+        "gulp": "^4.0.2",
+        "npm": "^7.7.5"
```

Comment:
revert?
```suggestion
    }
```

Reply:
Yea for some reasons it kep updating it an couldnt rollback unless i deleted the file.